src/pilot/client.patch

```diff
@@ -1,30 +1,45 @@
---- /usr/lib/python2.7/site-packages/dracclient/client.py	2019-02-15 20:14:23.863509901 +0000
-+++ client.py	2019-02-15 20:14:16.685378567 +0000
-@@ -272,7 +272,7 @@
-         state_reached = self._wait_for_host_state(
-             self.client.host,
-             alive=False,
--            ping_count=3,
```

Comment:
It looks like this patch needs to be updated as it appears to remove the patch that fixes iDRAC reset.  We can't remove this change that switches the ping count from 3 to 2 or iDRAC reset will break.

Reply:
Updated.